cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -47,6 +46,8 @@ import (
 
 const (
 	defaultEnvCfgFileName = "env-config.json"
+	LogTargetEventLog = "eventlog"
```

Comment:
Would consider moving these constants to the `register_event_logger_windows.go` or a separate constant files (same as [SSM Agent](https://github.com/aws/amazon-ssm-agent/blob/mainline/agent/appconfig/constants.go))

Reply:
Centralize constants is arguably a good idea, but I think organize them into a single constant class should be address in a separated "refactor" PR, here it's simply just following existing pattern for consistence.